packages/assets/projects/fallbacks/tests/vrt/specs/fallbacks.vrt.ts

```diff
@@ -1,9 +1,7 @@
 import { expect, test } from '@playwright/test';
+import { viewportWidths } from '@porsche-design-system/shared/testing/playwright.vrt.config';
 
-const fallbacks = ['cookies', 'browser-support'] as const;
-const viewportWidths = [320, 480, 760, 1000, 1300, 1760] as const;
-
-fallbacks.forEach((fallback) => {
+(['cookies', 'browser-support'] as const).forEach((fallback) => {
```

Comment:
as const assertion does nothing I guess.
```suggestion
(['cookies', 'browser-support']).forEach((fallback) => {
```

Reply:
It's not a big deal but might be helpful to have a typed fallback const in the loop.